pkgs/applications/editors/helix/default.nix

```diff
@@ -0,0 +1,25 @@
+{ fetchFromGitHub, lib, rustPlatform }:
+let
+  pname = "helix";
+  version = "0.0.10";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "helix-editor";
+    repo = pname;
+    rev = "v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-XyXlzm+fJyW+wg7XRealIvse+eO5AZw/6qprkq9ORV0=";
+  };
+
+  cargoSha256 = "sha256-Gm5ZEFATsHyzL/yVrXZb/neGcTdrSq55rfma4pxM2XA=";
+
```

Comment:
```suggestion
  checkFlags = [
    "--skip buffer::tests::index_of_panics_on_out_of_bounds" 
    "--skip buffer::tests::pos_of_panics_on_out_of_bounds"
    "--skip buffer::Buffer::pos_of"
    "--skip buffer::Buffer::index_of"
  ];
```

Reply:
we should figure out why these tests fail instead of disabling them

Reply:
I think this is because there were some warnings present, it passed otherwise. I'll cut a new release this weekend that should address this

Reply:
I tagged `0.2.0` today that should build fine.

Reply:
tried to build this pr, got failures on `buffer::tests::index_of_panics_on_out_of_bounds` and `buffer::tests::pos_of_panics_on_out_of_bounds`

Reply:
so the problem here is we're running checks on a release binary which doesn't enable debug assertions, so the tests that are expected to panic never do. i'm not sure it's feasible for nix to build the binary twice just for tests, so we need to either enable debug assertions via `RUSTFLAGS` or disable checks.